opensearch-maps-server/README.md

```diff
@@ -49,6 +50,16 @@ docker run \
     import
 ```
 
+Run the following command to import the tiles set from a local file:
+```
+docker run \
+    -e LOCAL_TILES=/tmp/planet-osm-default-z0-z8.tar.gz \
```

Comment:
The `LOCAL_TILES` is actually the path in the Docker container, not the host machine?